pages/common/ffmpeg.md

```diff
@@ -4,16 +4,20 @@
 
 - Extract the sound from a video and save it as MP3:
 
-`ffmpeg -i {{video_filename}} -vn -ar 44100 -ac 2 -ab 192 -f mp3 {{sound.mp3}}`
+`ffmpeg -i {{path/to/video}} -vn -ar 44100 -ac 2 -ab 192 -f mp3 {{sound}}.mp3`
 
 - Convert frames from a video or GIF into individual numbered images:
 
-`ffmpeg -i {{video_or_gif_filename}} {{image%d.png}}`
+`ffmpeg -i {{path/to/video_or_gif}} {{image%d.png}}`
 
 - Combine numbered images (image1.jpg, image2.jpg, etc) into a video or GIF:
 
-`ffmpeg -f image2 -i {{image%d.jpg}} {{video.mpg_or_video.gif}}`
+`ffmpeg -i {{path/to/image%d.jpg}} -f image2 {{video.mpg_or_video.gif}}`
```

Comment:
Nice change, the order is indeed more intuitive this way. :+1:

Reply:
Actually, on second thought, I'm unsure what `-f image2` means here. Additionally, we should probably use the pipe syntax `{{video.mpg|video.gif}}`.

Reply:
`-f image2` is ffmpeg's internal syntax. Even I don't know what its supposed to mean. I am guessing down the line, their image decoder changed, but they couldn't remove the old codecs due to compatibility, hence this weird flag. I could be totally wrong though.

Good suggestion on the pipe syntax.